.env

```diff
@@ -6,3 +6,14 @@ REACT_APP_BUILD_VERSION=
 # REACT_APP_TESTNET_API=https://testnet.nexus.stg.oasis.io/v1/
 REACT_APP_API=https://nexus.oasis.io/v1/
 REACT_APP_TESTNET_API=https://testnet.nexus.oasis.io/v1/
+# REACT_APP_NO_BUILD_BANNERS=true
+# REACT_APP_TITLE=Magic Explorer
+# REACT_APP_DESC=The official explorer for my cool project
+# REACT_APP_SOCIAL_TELEGRAM=https://t.me/oasisprotocolcommunity
+# REACT_APP_SOCIAL_TWITTER=https://twitter.com/oasisprotocol
+# REACT_APP_SOCIAL_DISCORD=https://oasis.io/discord
+# REACT_APP_SOCIAL_YOUTUBE=https://www.youtube.com/channel/UC35UFPcZ2F1wjPxhPrSsESQ
+# REACT_APP_SOCIAL_REDDIT=https://www.reddit.com/r/oasisnetwork/
+# REACT_APP_SOCIAL_LINKEDIN=https://www.linkedin.com/company/oasisprotocol
+# REACT_APP_SOCIAL_DOCS=https://oasisprotocol.org/developers#overview
+# REACT_APP_SOCIAL_HOME=https://oasisprotocol.org/
```

Comment:
At least `.env.production` should be set to this variables, otherwise it will break current functionality. Even `.env` should default to Oasis. Unless we use `.env` as template, and start using `.env.development` for development builds.

Reply:
>  Even `.env` should default to Oasis.

Done that now.

Reply:
> At least `.env.production` should be set to this variables, otherwise it will break current functionality. 

Why is that? As far as I understand, both `.env` and `.env.production` will be read and superimposed. Given that, why should we repeat values?

Reply:
https://www.npmjs.com/package/dotenv#should-i-have-multiple-env-files

> Avoid custom set ups that work in inheritance somehow (.env.production inherits values form .env for example). It is better to duplicate values if necessary across each .env.environment file.

Reply:
OK, duplicated values as suggested.

Reply:
ew

Reply:
I'm also not happy about the duplication, but apparently, this is indeed what the developers of the `.env` parsing system have envisioned.

I would much prefer a single source of truth, as opposed to synchronously updating duplicates....
... but I guess the tradeoff is that we don't have to remember who is inheriting what value from where, everything is always flat...

Reply:
We could do as I suggested in the first comment. Use `.env` as a template(default) values. But we should not use `.env` for building. And add `.env.development` for development builds.